resources/widgetlist.ts

```diff
@@ -322,35 +330,26 @@ class WidgetList extends HTMLElement {
         y = y + (this._yinc2 * this._elementheight);
         rowcolindex = 0;
       }
-    }
+    });
   }
 
   test() {
     for (let i = 0; i < 8; ++i) {
       const div = document.createElement('div');
-      div.style.width = this._elementwidth * 3 / 4;
-      div.style.height = this._elementheight * 3 / 4;
+      div.style.width = String(this._elementwidth * 3 / 4);
+      div.style.height = String(this._elementheight * 3 / 4);
       div.style.overflow = 'hidden';
-      div.style.backgroundColor = '#' + parseInt(Math.random() * 10) + '' + parseInt(Math.random() * 10) + '' + parseInt(Math.random() * 10);
+      div.style.backgroundColor = `#${Math.random() * 10}${Math.random() * 10}${Math.random() * 10}`;
```

Comment:
I think this is not the same without the `parseInt`.  Probably this was intended to be `Math.round()` or `Math.floor()`?